.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You can add this to your global .gitignore by putting it in `~/.gitignore` and then something like this in your `~/.gitconfig`: 

```
[core]
	excludesfile = ~/.gitignore
```